aspdotnet/tests/test_aspdotnet.py

```diff
@@ -0,0 +1,118 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import mock
+import os
+import pytest
+from datadog_checks.stubs import aggregator
+from datadog_checks.aspdotnet import AspdotnetCheck
+#import datadog_checks_tests_helper.datadog_test_libs.win
+import datadog_test_libs.win.pdh_mocks as mockpdh
+HERE = os.path.abspath(os.path.dirname(__file__))
+MINIMAL_INSTANCE = {
+    'host': '.',
+}
+
+INSTANCE_WITH_TAGS = {
+    'host': '.',
+    'tags': ['tag1', 'another:tag']
+}
+
+
+@pytest.fixture
+def Aggregator():
+    aggregator.reset()
+    return aggregator
+
+
+class TestASPDotNet:
```

Comment:
I think we want to remove this class. It doesn't really do anything since you're not maintaining any state here, all it would change is the indent and removing the `self.`